### ISSUE TITLE
CHANGELOG missing

### ISSUE THREAD

**Issue by isantolin** [arduino/openwrt-yun-1505#1]
this repository doesn't have a Changelog file

**Comment by facchinm** [comment]
Correct, we just opened the repo so things are setting up, the build instructions are obscure and the difference between `openwrt-yun` (based on Linino) and `openwrt-yun-1505` (based on OpenWrt vanilla) are not stressed anywhere.
But we'll set things up in the next few hours/days so don't worry :wink: 
Thanks for reporting

**Comment by impressiver** [comment]
Might be worth a blog post explaining what's happening behind the scenes. I was getting started with Arduino Cloud tonight, hadn't noticed the first official release of the Yun firmware (since 2014) dropped last week, and things started snowballing from there.

Arduino Cloud requires 1.6.2, but the Arduino docs state that updating to 1.6.2 from 1.5.3 (which was the standard download until last week) will void the warranty. However, not doing the whole TFTP flash (?!) dance will result in a bricked device.

So I ran to Google and Github for some clarity, only to find a whole new Yun build with no explanation. I figured it out from the mentions of `OpenWRT`, but I have no clue what I'll end up with after this goose chase.

**Comment by niallp** [comment]
Currently trying a complete build via build.sh, might work though noted the script includes copying some keys from home directory I don't have ... are these supposed to be generated locally or downloaded from somewhere else ? 

Any additional build instructions would be helpful. arduino.cc is explicit that updated u-boot is required to get 1.6.2 installed yet doesn't provide any binary for it (just a sysupgrade.bin)

**Comment by facchinm** [comment]
Hi @niallp ,
the u-boot patch is here https://github.com/arduino/openwrt-yun/commit/c9c80c6afa262e467d56f06419b56bd5cb2af740 . In fact, you can download the binary from http://downloads.arduino.cc/openwrtyun/1.6.2/openwrt-ar71xx-generic-linino-u-boot.bin ; I'll double check the website pages to recover the link if it disappeared during an edit.

**Comment by isantolin** [comment]
@facchinm any chance to update version to 15.05.1

**Comment by niallp** [comment]
@facchinm , thanks for the link to the binary, my build from git has died twice so something else to sort out there :-/ ... I guessed at the links for the corresponding kernel and rootfs and got them downloaded so in theory now have a matched set for 1.6.2 (would be nice if the download server allowed browsing, especially when the webpages don't keep up !)

board seems to be running fine, directions on advanced update page were all OK except it should be rebooted via reset switch instead of bootm command (otherwise the kernel boot hangs)

u-boot version: U-Boot 1.1.4-g80b2b7c3-dirty (Feb 22 2016 - 11:48:17)

kernel version :
Linux version 3.18.23 (admin@compile-01-prod) (gcc version 4.8.3 (OpenWrt/Linaro GCC 4.8-2014.04 r48749) ) #34 Tue May 10 08:20:13 UTC 2016

rootfs version:
BusyBox v1.23.2 (2016-03-23 22:30:11 UTC) built-in shell (ash)

---

 |       |.-----.-----.-----.|  |  |  |.----.|  |_
 |   -   ||  _  |  -**|     ||  |  |  ||   _||   _|
 |_____**||   **|___**|**|**||________||**|  |__**|
          |__| W I R E L E S S   F R E E D O M

---

 CHAOS CALMER (1.6.2, r48749) - Arduino Yun101 and Yun Shield
